### PR TITLE
i hope it fixes cleaning lvm metadata

### PR DESCRIPTION
**Pull request by kozhukalov**

### CODE REVIEW

**Comment by xdel** [comment]
I have such hope too. +1

File: deployment/puppet/cobbler/templates/scripts/pmanager.py [hunk 1]
```diff
@@ -630,24 +664,23 @@ def lv(self):
                 #                 self.unit,
                 #                 end_size,
                 #                 disk["size"]))
+
+                self.log_lvm("before hdparm id={0}".format(disk["id"]), False)
                 self.late("sleep 3")
                 self.late("hdparm -z $(readlink -f /dev/{0})".format(disk["id"]))
-                self.late("pvcreate $(readlink -f /dev/{0}){1}".format(disk["id"], pcount))
+                self.late("dd if=/dev/zero of=$(readlink -f /dev/{0}){1} bs=1M count=64".format(disk["id"], pcount))
+                self.late("mkfs.xfs -q $(readlink -f /dev/{0}){1}".format(disk["id"], pcount))
```
Comment (aglarendil) [inline deployment/puppet/cobbler/templates/scripts/pmanager.py:672]: wtf?

Comment (xdel) [inline deployment/puppet/cobbler/templates/scripts/pmanager.py:672]: "Fake" FS creation. I have no idea how it helps. And yes, entire ubuntu-related pmanager is a kinda wtfish.

**Comment by aglarendil** [comment]
+2

**Pull request merged**